app/src/auto_validator/core/admin.py

```diff
@@ -62,18 +47,14 @@ class SubnetSlotAdmin(admin.ModelAdmin):
         "subnet",
         "blockchain",
         "netuid",
-        "registered",
-        "maximum_registration_price",
-        "restart_threshold",
-        "reinstall_threshold",
+        "is_registered",
+        "max_registration_price_RAO",
         "registration_block",
         "deregistration_block",
     )
     search_fields = ("subnet__name", "netuid")
     list_filter = ("blockchain",)
-
-    def registered(self, obj):
-        return obj.registered_status()
+    list_per_page = 2
```

Comment:
This was only for testing, doesn't make sense to commit it to the repository.

Self-review should have caught it.

Reply:
From my end, I do not have any code related to that issue.